randovania/layout/preset_migration.py

```diff
@@ -945,6 +945,21 @@ def _migrate_v59(preset: dict) -> dict:
 
     return preset
 
+def _migrate_v60(preset: dict) -> dict:
+    config = preset["configuration"]
+    game = preset["game"]
+
+    if game in {"dread"}:
+        config["elevators"] = {
+            "mode": "vanilla",
+            "excluded_teleporters": [
+            ],
+            "excluded_targets": [],
+            "skip_final_bosses": False,
```

Comment:
yuck. I know it's built into the base class but ugh

Reply:
Is this a comment for future™ or do we want to refactor that now?

Reply:
up to you

Reply:
I'd like for the field to be removed from the base class in this case.